src/test/resources/csharp-file.cs

```diff
@@ -0,0 +1,538 @@
+﻿using ast_visual_studio_extension.CxWrapper.Exceptions;
+using ast_visual_studio_extension.CxWrapper.Models;
+using log4net;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows.Navigation;
+
+namespace ast_visual_studio_extension.CxCLI
+{
+    public class CxWrapper
```

Comment:
what is this file? is this file the test file but it's the wrapper .cs?
maybe we can have a simpler file (or 2 files one. with a real vulnerability, only one, and other one that will return an err?)